lib/registration.pm

```diff
@@ -776,7 +776,7 @@ sub registration_bootloader_cmdline {
     set_var('SCC_URL', 'https://scc.suse.com') unless get_var('SCC_URL');
     my $cmdline = '';
     if (my $url = get_var('SMT_URL') || get_var('SCC_URL')) {
-        $cmdline .= is_agama ? " inst.register_url=$url" : " regurl=$url";
+        $cmdline .= " regurl=$url";
```

Comment:
I think we still need set inst.register_url  for agama, in VR this passed may for the scc.suse.com is by default, please run a VR without regurl set to prove that.
Could we change the condition to:  || !(get_var('SCC_URL') =~ 'scc.suse.de') ?  And do not change the $cmdline.

Reply:
But set this will only work in development group, in production group we need to register proxyscc via inst.register_url in production group, (in development group we don't need it since register to SCC) So for current situation, I think we need adjust test process, when inst.register_url set we need select 'provide registration code' to show registration code field to fill registration code. And we need to merge it only after the feature released to RC3.